### ISSUE TITLE
Tooltip takes a few milliseconds to appear

### ISSUE THREAD

**Issue by KumarAbhirup** [michelson/dante2#165]
Try selecting some text and immediately press delete button. You'll see the glitch that it won't erase the selected text. In fact, the editor does not work before tooltip appears.

When you select text by keyboard (by pressing Shift+Arrow Keys), tooltip appears quickly and editor works without glitch.

But if you select by mouse, Dante2 makes you wait 900 milliseconds to see the tooltip first and only then act on the editor.

**This is a serious issue. Needs a fix.**

I guess there is some `setTimeout()` function in codebase for tooltip to appear after selecting text.

**Comment by michelson** [comment]
Hi @KumarAbhirup , it's not clear to me how the issue is. Can you make a screen record on the issue to dig further?

thanks for report this!

**Comment by michelson** [comment]
could not replace it http://g.recordit.co/vhpwQAj0Rx.gif

**Comment by KumarAbhirup** [comment]
Sure 👍 
Here's the [video](https://www.dropbox.com/s/1bvn8ngy4jn8tra/dante2%20bug.mov?dl=0) hosted on dropbox.
Same bug at [demo page](https://michelson.github.io/dante2/#/) too.

**Comment by michelson** [comment]
Hi @KumarAbhirup ,
it's supposed to work like that. when you select with mouse the tooptip will only appear when you release the mouse (mouseup)

**Comment by michelson** [comment]
here is a demo that demonstrate that on medium.com
it's the same functionality http://recordit.co/LaRVjQJRXZ

**Comment by KumarAbhirup** [comment]
Tooltip's late appearance is not at all a problem. But the text selected cannot be deleted before tooltip appears. It's a small glitch that seldom appears. But for super mouse users like me, I feel the bug everytime I try to delete some text by selecting it.

**Comment by michelson** [comment]
it seems that is a problem on https://draftjs.org/ core . could you try to
replicate it in order to confirm that ?

thanks

**Comment by KumarAbhirup** [comment]
Yes, I was able to reproduce the issue. If it's a draftjs issue, could we do something to fix this bug?

**Comment by KumarAbhirup** [comment]
I guess you can close this issue, since it is a draft-js problem and not a dante2 problem.
